libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.ts

```diff
@@ -132,15 +132,15 @@ export class SkyColorpickerComponent
   /**
    * A help key that identifies the global help content to display. When specified, a [help inline](https://developer.blackbaud.com/skyux/components/help-inline)
```

Comment:
```suggestion
   * A help key that identifies the global help content to display. When specified along with `labelText`, a [help inline](https://developer.blackbaud.com/skyux/components/help-inline)
```
In some existing property descriptions, we include "along with `labelText" in the opening clause of the second sentence. I think it's worth keeping those instances and adding that detail to the descriptions that don't have it in addition to the new sentence at the end. This is more consistent, and it also indicates the dependency closer to the beginning of the descriptions so that users are hopefully less likely to overlook it.